lib/scheduler.js

```diff
@@ -71,6 +73,45 @@ exports.scheduler = {
         }, destroyTime);
     },
     /**
+     * Schedules the next notification based on our current step.
+     */
+    scheduleNextNotification: function() {
+        let step = storageManager.get('step');
+
+        // clear any currently scheduled notification timer
+        module.exports.scheduler.clearNotificationTimer();
+        // start autoCloseTimer
+        module.exports.scheduler.autoCloseTimer(intervals.afterInteractionCloseTime);
```

Comment:
Nit: It's a little long-winded and feels a bit weird to be calling functions directly on the back of `module.exports`. Could the export definition be moved to the end as opposed to defined directly inline?